site-src/geps/gep-1709.md

```diff
@@ -37,8 +45,295 @@ which implementations can prove they satisfy and be recognized for.
+The following extended features were not supported:
+- HTTPResponseHeaderModification
+- HTTPRouteQueryParamMatching
+```
+
+For the above output upstream Gateway API can be provided the following report:
+
+```json
+kind: ConformanceReport
+implementation: acmeorg-acme
+profiles:
+- name: Layer7Extended
+  successes:
+  - tag: v0.7.0
+    unsupportedFeatures:
+    - HTTPResponseHeaderModification
+    - HTTPRouteMethodMatching
+  - tag: v0.6.2
+    unsupportedFeatures:
+    - HTTPResponseHeaderModification
+    - HTTPRouteMethodMatching
+    - HTTPRouteQueryParamMatching
+  - tag: v0.6.1
+    unsupportedFeatures:
+    - HTTPResponseHeaderModification
+    - HTTPRouteMethodMatching
+    - HTTPRouteQueryParamMatching
+maintainers:
+- @kubernetes-sigs/gateway-api-maintainers
+```
+
+> **NOTE**: In the above the `implementation` field is a combination of
+> `<organization>-<project>`. Organizations can be an open source organization,
+> an individual, a company, e.t.c.. Organizations can theoretically have more
+> than one `<project>` name and submit separate reports for each of them.
+
+> **NOTE**: The above assumes that the `acmeorg-acme` project was already
+> passing for versions prior to `v0.7.0` and that the new passing version was
+> added to the top of the list. This allows implementations to report their
+> historical passes which will become relevant as we get into future major
+> releases. You can also see that there's been progression to add the
+> `HTTPRouteQueryParamMatching` feature this release.
+
+> **NOTE**: The `maintainers` field indicates the Github usernames or team
+> names of those who are responsible for maintaining this file, so they can be
+> easily contacted when needed (e.g. for relevant release announcements
+> regarding conformance, e.t.c.).
+
+The implementation can compile its `ConformanceReport` and upload it to the
+Gateway API project by creating a pull request. The file should be uploaded
+to the following sub-directory:
+
+```console
+conformance/results/<organization>-<project>.yaml
+```
+
+> **NOTE**: the first time an implementation adds their report, they need
+> to also add the Golang templating to compile that report into their section
+> of the implementations page if that's not already been handled.
+
+This will start the certification process which will result in the
+implementation receiving a badge they can use on their project pages and/or
+repositories which indicates their conformance levels and link to their
+conformance reports in the upstream Gateway API website.
+
+> **NOTE**: No verification process (to prevent intentionally incorrect
+> conformance results) will be implemented at this time. We expect that this wont
+> be an issue in our community and even if someone were to try and "cheat" on
+> the reporting the reputation loss for being caught would make them look very
+> bad and would not be worth it.
+
+#### Certification Process
```

Comment:
for this iteration, should we consider any kind of visualisation of conformance reports and/or badges other than looking into `conformance/results/<organization>-<project>.yaml` files?

Reply:
I think its better to keep it for future iterations but maybe worth adding a reference here

Reply:
I think that would be good, however I'm not sure _what_ that should look like yet. I'm actually hoping that someone else will swoop in and think up a display layer using the raw data laid out so far.

Reply:
I added a TODO comment to the document that says we must figure this out _BEFORE_ we consider this `implementable`. That way we don't have to hold up this iteration just for that bit. :+1: